File: python vs cpp/4_List python vs  Array cpp/6_sum_of_array.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;  // Input the size of the array

    int ar[n];  // Declare an array of size 'n'

    // Input 'n' values into the array
    for (int i = 0; i < n; i++) {
        cin >> ar[i];
    }

    int sum = 0;
    // Calculate the sum of the array elements
    for (int i = 0; i < n; i++) {
        sum += ar[i];
    }

    // Output the sum
    cout << sum;

    return 0;
}

File: 0_python compare to  cpp/4_List python vs  Array cpp/6_sum_of_array.py
# Input the size of the list
n = int(input())

# Declare a list and input 'n' values
ar = []
for i in range(n):
    ar.append(int(input()))

# Calculate the sum of the list elements
sum_ar = sum(ar)

# Output the sum
print(sum_ar)
